test/functorch/test_control_flow.py

```diff
@@ -5,13 +5,43 @@
 from torch.testing._internal.common_utils import TEST_WITH_TORCHDYNAMO
 import torch
 import torch.utils._pytree as pytree
-from torch._functorch.aot_autograd import from_fun, to_fun
 from functorch.experimental import control_flow
 from functorch.experimental.control_flow import UnsupportedAliasMutationException, cond
 from torch.fx.experimental.proxy_tensor import make_fx
 from torch.testing._internal.common_utils import run_tests, TestCase
 from torch._dynamo.exc import CondOpArgsMismatchError
 from torch.testing._internal.common_quantization import skipIfNoDynamoSupport
+from torch._subclasses.functional_tensor import FunctionalTensor
+
+# TODO: pull these helpers from AOTAutograd later
```

Comment:
Needs some deduping

Reply:
Yep - I'm definitely cluttering up the tests will multiple copies of these floating around.

Are you ok with me doing the cleanup in https://github.com/pytorch/pytorch/pull/106406? (this feels like less churn to me, since I think that long term it's reasonable for AOTAutograd to be the source of truth for these API's anyway)

Reply:
Yeah, that sgtm